Orange/widgets/data/tests/test_owconcatenate.py

```diff
@@ -21,6 +21,12 @@ def setUp(self):
         self.iris = Table("iris")
         self.titanic = Table("titanic")
 
+    def test_no_input(self):
+        self.widget.apply()
+        self.widget.controls.append_source_column.toggle()
+        self.widget.apply()
```

Comment:
The call to `apply` could be suppressed in input setters and/or handleNewSignals, but still triggered by a click on the 'Apply' button in the widget.